File: events_schedule_category.js
const url = 'https://os-sports-perform.p.rapidapi.com/v1/events/schedule/category?category_id=26&date=2022-05-11';
const options = {
	method: 'GET',
	headers: {
		'x-rapidapi-host': 'os-sports-perform.p.rapidapi.com'
	}
};

try {
	const response = await fetch(url, options);
	const result = await response.text();
	console.log(result);
} catch (error) {
	console.error(error);
}

File: events_schedule_category.py
# sofascore-api

import requests

url = "https://os-sports-perform.p.rapidapi.com/v1/events/schedule/category"

querystring = {"category_id":"26","date":"2022-05-11"}

headers = {"x-rapidapi-host": "os-sports-perform.p.rapidapi.com"}

response = requests.get(url, headers=headers, params=querystring)

print(response.json())
